### PR TITLE
Adds init command

### LINKED ISSUE #192: init command to create a default config file
fixed by #191

### PR DESCRIPTION
**Pull request by micheleorselli**
fixes #192

### CODE REVIEW

File: src/CLI/Command/Init.php [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                '.'
+            );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $output->writeln('');
+
+        try {
+            $sourceFilePath = __DIR__.'/../../../phparkitect-stub.php';
```
Comment (AlessandroMinoccheri) [inline src/CLI/Command/Init.php:46]: Have you tried launching the command with `.phar`?
In past, we had problems with path when we try to run a command with our different installations: `composer` and `.phar`

Comment (micheleorselli) [reply src/CLI/Command/Init.php:46]: Nope, i'll give it a try

Comment (micheleorselli) [reply src/CLI/Command/Init.php:46]: yup, phar generation needed a fix, everything should be fine now

**Review by AlessandroMinoccheri — APPROVED** [review]
LGTM 👍

**Pull request merged**